include/flatcc/portable/pdiagnostic_pop.h

```diff
@@ -1,15 +1,25 @@
 #ifndef PDIAGNOSTIC_POP_H
 #define PDIAGNOSTIC_POP_H
 
+#if defined(PDIAGNOSTIC_PUSHED_MSVC)
```

Comment:
This is not necessary, because the flag is always defined in the corresponding push logic, and likewise for the following code.